pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -169,10 +169,10 @@ type SubscriptionsAPIFilter struct {
 	// +optional
 	Suffix map[string]string `json:"suffix,omitempty"`
 
-	// SQL is a CloudEvents SQL expression that will be evaluated to true or false against each CloudEvent.
+	// CESQL is a CloudEvents CESQL expression that will be evaluated to true or false against each CloudEvent.
 	//
 	// +optional
-	SQL string `json:"sql,omitempty"`
```

Comment:
I like this proposal, and since we touch an _alpha stage_ experimental feature, this seems right to do.

Makes it more explicit, rather than just a generic `sql` field on the API :+1: